app/views/users/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<h2><%= t ".list" %></h2>
+<table>
+  <% @users.each do |user| %>
+    <tr>
+      <td>
+        <%= user.id %>
+      </td>
+      <td>
+        <%= user.name %> --
```

Comment:
What is "--" here? :)

Reply:
Sorry, this make a easy sense for me. My team have another one make the interfaces